web/apps/labelstudio/src/app/App.scss

```diff
@@ -9,6 +9,11 @@ body {
 
   margin: 0;
   font-family: Figtree, sans-serif;
+  display: flex;
+}
+
+:global(.app-wrapper) {
+  flex: 1;
 }
 
 .app-wrapper {
```

Comment:
not sure this is actually applying to anything - anyone against removing this block?